spec/no_duplicate.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+
+git clone https://github.com/puppetlabs/puppetlabs-stdlib
+
+stdlib_funcs=`ls puppetlabs-stdlib/lib/puppet/parser/functions`
+extlib_funcs=`ls lib/puppet/parser/functions`
+
+for func in $extlib_funcs
+do
+    if `echo $stdlib_funcs | grep $func >/dev/null`; then
```

Comment:
This pattern is too generic. You're doing but a substring match here, where you want exact matches, no?